### DISCUSSION TITLE
docs/blog

### DISCUSSION THREAD

**Discussion by giscus** [flowershow/flowershow#530] [Announcements]
# docs/blog

All files inside the \<your-content-folder>/blog will be treated as blog posts.Flowershow will display authors from the authors frontmatter list field below the blog title. It will also try to link ea...

https://flowershow.app/docs/blog

**Comment by rufuspollock** [comment]
Can I have blog posts outside of the blog folder? And, if so, what must i add to frontmatter of the note to make it a blog post?

**Reply by everythingness** [reply]
yes! i'd like the blogposts to be on the root level of my content directory if that was possible. i don't need a separate blog directory, nor do i need a separate index page for the site. the index page can just be the blogs page.

**Reply by 1001Josias** [reply]
Have you had success with this?
